libs/k8s/custom/core/volumeMounts.libsonnet

```diff
@@ -43,12 +43,12 @@ local d = import 'doc-util/main.libsonnet';
         )
         else {}
       );
+      local volumeMixins = [volume.fromConfigMap(name, name) + volumeMixin];
 
       super.mapContainers(addMount) +
```

Comment:
This is just one occurrence, but I think you're missing to pass the `includeInitContainers` argument here

Reply:
Good spot! Should be fixed.